charts/cloudzero-agent/values.yaml

```diff
@@ -62,12 +60,11 @@ imagePullSecrets: []
 validator:
   serviceEndpoints:
     kubeStateMetrics:
-    prometheusNodeExporter:
   # -- Flag to skip validator failure if unable to connect to the CloudZero API.
   name: env-validator
   image:
     repository: ghcr.io/cloudzero/cloudzero-agent-validator/cloudzero-agent-validator
-    tag: 0.4.1
+    tag: 0.8.0
```

Comment:
are we sure that we want this? we pinned to 0.4.1 because some changes after that didn't work with this version of the chart. I assume that's resolved in 0.8.0?

Reply:
Sorry I'm late to the party here, but for posterity, Dan's right. Changes after `0.4.1` were related to Dynamic Service Discovery of KSM. We ended up moving away from DSD since we no longer allow users to bring their own KSM, therefore we can directly tell that validation the KSM endpoint. Long story short, this should be fine since we don't directly call the Validator's `generate` command, which would have been called by the chart had we moved forward with DSD.